### ISSUE TITLE
Focus border not visible in regular themes

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52405]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
Keyboard,Usability

**Repro Steps:**
1.Navigate to "Activity Bar" at left side of the screen after login.
2.Navigate to any item such as "Explorer,Search,SourceControl,Debug,Extensions,Setting" buttons.
3.Clicked on any one mentioned button and through mouseover move the mouse to any one remaining button.
3.Observed that there is same focus is visible to user.

**Actual:**
After clicking on "Explorer" button then using tab/mouseover move to other button list except "Explorer" then it shows same focus color to each becuase of this user will not able to differentiate which button is selected.

**Expected:**
There should be some difference between the selected and non selected item. if explorer is selected then it should be dark and if other buttons are not selected then it should not look like selected.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets.
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**User Impact:**
User will not able to differentiate the selected and non selected button from activity bar.

**Attachment For Reference:**
[[Accessibility][Usability]A11y_VSCode_ActivityBar_Keyboard.pptx](https://github.com/Microsoft/vscode/files/2118221/Accessibility.Usability.A11y_VSCode_ActivityBar_Keyboard.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by bpasero** [comment]
High contrast theme for reference:

![image](https://user-images.githubusercontent.com/900690/42096703-5a57fece-7bb6-11e8-9d8e-c20dd45fa617.png)

**Comment by isidorn** [comment]
This is as designed for now. And we have no plans to support this in non high contrast themes.
Every theme author can add this, thus it is up to extensions to support this if they want to.
